src/XCCDF_POLICY/xccdf_policy_remediate.c

```diff
@@ -1143,10 +1143,23 @@ static int _write_script_header_to_fd(struct xccdf_policy *policy, struct xccdf_
 	} else if (oscap_streq(sys, "urn:redhat:osbuild:blueprint")) {
 		char *blueprint_fix_header = oscap_sprintf(
 			"%s"
-			"name = \"%s\"\n"
+			"name = \"hardened_%s\"\n"
 			"description = \"%s\"\n"
-			"version = \"%s\"\n",
-			fix_header, profile_id, profile_title, benchmark_version_info);
+			"version = \"%s\"\n\n"
+			"[customizations.openscap]\n"
+			"profile_id = \"%s\"\n"
+			"# If your hardening data stream is not part of the 'scap-security-guide' package\n"
+			"# provide the absolute path to it (from the root of the image filesystem).\n"
+			"# datastream = \"/usr/share/xml/scap/ssg/content/ssg-xxxxx-ds.xml\"\n\n"
+			"# If your hardening data stream is not part of the 'scap-security-guide' package\n"
+			"# you don't need this package to be installed in the image (section can be removed).\n"
+			"[[packages]]\n"
+			"name = \"scap-security-guide\"\n"
+			"version = \"*\"\n\n"
+			"[[packages]]\n"
+			"name = \"openscap-scanner\"\n"
+			"version = \"*\"\n\n",
```

Comment:
I probably wouldn't do this.

The target system doesn't need `scap-security-guide` or `openscap-scanner` since the changes were already applied during image creation. It might be convenient in case the admin wants to run `oscap` from CLI to double-check, but it unnecessarily bloats the image.

Plus it might provide confusing results if the admin used `datastream = ` for `[customizations.openscap]` since the installed `scap-security-guide` RPM will likely have different content version.

Reply:
I've removed these sections. By the time it hits downstream they won't be needed (they are implicit in the upstream of IB now).